app/Exceptions/Handler.php

```diff
@@ -28,6 +29,10 @@ public function report(Exception $exception): void
 
     public function render($request, Exception $exception): Response
     {
+        if($exception instanceof NotFoundHttpException) {
```

Comment:
I don't think this is necessary for Laravel 5.7, I think you can safely revert this whole file.

[Custom Error Messages](https://laravel.com/docs/5.7/errors#custom-http-error-pages)